tests/plugins/kernel_install/utils_test.sh

```diff
@@ -68,6 +68,28 @@ function test_ask_yN()
   assert_equals_helper 'TEST_MODE' "$LINENO" '0' "$output"
 }
 
+function test_ask_with_default()
+{
+  output=$(printf '%s\n' 'something' | ask_with_default 'Test message' 'lala')
+  assert_equals_helper 'TEST_MODE' "$LINENO" 'something' "$output"
```

Comment:
The first parameter of assert_equal_helper should be a message that a developer can read and understand.